packages/devops/scripts/deployment/deployPackages.sh

```diff
@@ -67,7 +49,7 @@ for PACKAGE in ${PACKAGES[@]}; do
         yarn migrate
 
         # After running migrations it's good to ensure that the analytics table is fully built
-        yarn download-parameter-store-env-vars --package-name data-api --environment $ENVIRONMENT
+        yarn download-env-vars $BRANCH data-api
```

Comment:
See other note about extracting the list of packages that have .env vars out to a script. If we do that, we don't need a special case here for data-api, all packages would have .env vars added at the top of this script.

Reply:
True. Ok getting closer to sold on this!